src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="container">
+    <div class="calendar">
+      <div class="calendar__day-1 calendar__day-1--start-day-wed"></div>
+      <div class="calendar__day-2"></div>
```

Comment:
Do not add classes for each day, as they are sharing the same styles you can add sipmply **calendar__day**